src/main/java/com/sonymobile/tools/gerrit/gerritevents/dto/GerritChangeStatus.java

```diff
@@ -0,0 +1,47 @@
+package com.sonymobile.tools.gerrit.gerritevents.dto;
+
+/**
+ Contains constants that represent the status of changes.
+ */
+public enum GerritChangeStatus {
+    /**
+     * Change is open.
+     */
+    NEW("NEW"),
+    /**
+     * Change was merged.
+     */
+    MERGED("MERGED"),
+    /**
+     * Change was abandoned.
+     */
+    ABANDONED("ABANDONED"),
+    /**
+     * Catch-all type if Gerrit adds a new ChangeStatus we don't know about.
+     */
+    UNKNOWN("UNKNOWN");
+
+    private final String status;
```

Comment:
_nit_ The status is a bit redundant since `GerritChangeStatus.name()` will give the same value.

Reply:
yes, you are right